portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro

```diff
@@ -417,6 +417,15 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	macro gotoEllipsisMenuItemInItemSelector {
+		Click(
```

Comment:
You can reuse `Click(locator1 = "AssetLists#ASSET_LIST_ELLIPSIS")` instead of adding a new path for that. Like `AssetListsAdmin.viewContent` macro.